Gemfile.lock

```diff
@@ -205,8 +206,11 @@ GEM
       actionpack (>= 4.0)
       activesupport (>= 4.0)
       sprockets (>= 3.0.0)
+<<<<<<< Updated upstream
     sqlite3 (1.4.2)
     temple (0.8.2)
+=======
+>>>>>>> Stashed changes
```

Comment:
It looks like you've got unresolved conflicts in this file. Please resolve them accordingly.